docs/content/integrations/airflow/reference.mdx

```diff
@@ -0,0 +1,256 @@
+        repostitory_location_name="example_location",
+        repository_name="my_dagster_project",
+        job_name="all_assets_job",
+    )
+```
+
+In airflow 2.0+ you can create a dagster connection type to store configuration related to your dagster cloud organization, you can also pass this directly to the operator if you're using airflow 1.0.
+
+## Exporting Dagster jobs to Airflow
+
+You can compile Dagster jobs into DAGs that can be understood by Airflow. Each op in the job becomes an Airflow task. For example, here's a Dagster job:
+
+```python file=/integrations/airflow/hello_cereal.py
+import csv
+
+import requests
+
+from dagster import job, op
+
+
+@op
+def download_cereals():
+    response = requests.get("https://docs.dagster.io/assets/cereal.csv")
+    lines = response.text.split("\n")
+    return [row for row in csv.DictReader(lines)]
+
+
+@op
+def find_sugariest(context, cereals):
+    sorted_by_sugar = sorted(cereals, key=lambda cereal: cereal["sugars"])
+    context.log.info(f'{sorted_by_sugar[-1]["name"]} is the sugariest cereal')
+
+
+@job
+def hello_cereal_job():
+    find_sugariest(download_cereals())
+```
+
+To make this job available inside Airflow, you can write an Airflow DAG definition file that invokes <PyObject module="dagster_airflow" object="make_airflow_dag" />:
+
+```python file=/integrations/airflow/hello_cereal_dag.py
+# pylint: disable=unused-variable
+
+import datetime
+
+from dagster_airflow.factory import make_airflow_dag
+
+DEFAULT_ARGS = {
+    "owner": "airflow",
+    "depends_on_past": False,
+    "start_date": datetime.datetime(2019, 11, 7),
+    "email": ["airflow@example.com"],
+    "email_on_failure": False,
+    "email_on_retry": False,
+}
+
+dag, tasks = make_airflow_dag(
+    module_name="docs_snippets.integrations.airflow.hello_cereal",
+    job_name="hello_cereal_job",
+    dag_kwargs={"default_args": DEFAULT_ARGS, "max_active_runs": 1},
+)
+```
+
+If you run this code interactively, you'll see that `dag` and `tasks` are ordinary Airflow objects, just as you'd expect to see when defining an Airflow pipeline manually:
+
+```python
+>>> dag
+<DAG: hello_cereal_job>
+>>> tasks
+[<Task(DagsterPythonOperator): hello_cereal>]
+```
+
+Like other Airflow DAG definition files, this should go inside `$AIRLFLOW_HOME/dags`. The `docs_snippets.integrations.airflow.hello_cereal` module that's passed as the value for the `module_name` argument must be importable via the sys.path.
+
+After this, the DAG should show up inside Airflow:
+
+<Image
+alt="intro_airflow_one.png"
+src="/images/deploying/intro_airflow_one.png"
+width={1272}
+height={868}
+/>
+
+### Running Containerized
+
+The approach above runs each op inside an operator that's similar to the Airflow PythonOperator. If you instead want to containerize your Dagster job and run it using an operator that's similar to the Airflow DockerOperator, you can use <PyObject module="dagster_airflow" object="make_airflow_dag_containerized" />.
```

Comment:
```suggestion
The approach above runs each op inside an operator that's similar to the Airflow `PythonOperator`. If you instead want to containerize your Dagster job and run it using an operator that's similar to the Airflow `DockerOperator`, you can use <PyObject module="dagster_airflow" object="make_airflow_dag_containerized" />.
```

Formatting these guys as code for consistency since it's done in a later paragraph